package.json

```diff
@@ -82,6 +82,7 @@
 		"sway": "=1.0.0",
 		"valid-url": "=1.0.9",
 		"wamp-socket-cluster": "=2.0.0-beta.4",
+		"winston": "=3.0.0",
```

Comment:
@shuse2 Based on discussion in issue, any specific reason you choose `winston`. Its the slowest logger in all benchmarks. 

I still believe `bunyan` is better option for us, as it give good set of features for CLI usage that is helpful for us in our project scope. e.g. filtering logs based on some conditions. See https://github.com/trentm/node-bunyan#cli-usage

If you choose `winston` for profiling, then I guess its not the right choice. Logging and profiling are both totally different things. We should consider these different. For logging our choice should be based on logging features. When we need profiler, we will search some tool/lib specialized in profiling.

Reply:
Since console output is for mainly development purpose, I think it's more simpler and useful if it's just console log when it's turned on without external tool.

If we only consider performance, probably `pino`, but if we consider feature it would be `winston`.
I chose `winston` based on the fact that
- feature rich (formatting, profiling, container)
- 3.0 just released, so active development
- console logging by default

Reply:
Also, yes winston is slower than bunyan, but
```
benchBunyan*10000: 737.315ms
benchWinston*10000: 766.665ms
benchPino*10000: 306.935ms
```
i dont think it's huge difference to be considered